devel/kubecolor/Portfile

```
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/kubecolor/kubecolor 0.5.0 v
+go.offline_build    no
+revision            0
+
+categories          devel
+maintainers         {semihbkgr @semihbkgr} openmaintainer
+license             MIT
+
+description         \
+    Kubecolor is a tool that colorizes the output of kubectl commands to \
+    improve readability.
+long_description    {*}${description} \
+    \nKubecolor is a command-line tool that improves the readability of \
+    Kubernetes command outputs by adding color to kubectl results. By \
+    highlighting important information, it makes output easier to interpret, \
+    helping users manage Kubernetes resources more efficiently and enhancing \
+    overall workflow productivity.
+
+checksums           rmd160  3a6db5c54c3c2cffba409a7df116eaea16309a4c \
+                    sha256  251b3e2e84bd3574a9c628961066c8b41c403de6ecfb83b7aebd0dd5d7018290 \
+                    size    1368381
+
+build.args          -o ${name} -ldflags "-X main.Version=${version}" .
```

Thank you for making these changes, @semihbkgr. We actually need to make a small tweak here and escape the double quotes:

```suggestion
build.args          -o ${name} -ldflags \"-X main.Version=${version}\" .
```